src/redux/auth/operations.js

```diff
@@ -0,0 +1,33 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import * as api from "../../service/apiService";
+
+export const register = createAsyncThunk(
+  "auth/register",
+
+  async (data, thunkAPI) => {
+    try {
+      const response = await api.register(data);
+      console.log(response);
```

Comment:
try to log in console only neccessary info, so that not to miss smth important among all messages, especially long responses

Reply:
ok. thanks.